nengo/tests/test_dists.py

```diff
@@ -79,6 +79,28 @@ def test_hypersphere_surface(dimensions, rng):
     assert np.allclose(np.mean(samples, axis=0), 0, atol=0.25 / dimensions)
 
 
+@pytest.mark.parametrize("low,high", [(-2, -1), (-1, 1), (1, 2), (1, -1)])
```

Comment:
There's no formal check, it just makes them 0 if you give numbers < 0. Do you think it will be a better idea to throw a value error instead?